packages/cosmic-swingset/lib/launch-chain.js

```diff
@@ -154,7 +154,7 @@ export async function launch(
   }
 
   const [savedHeight, savedActions] = JSON.parse(
-    storage.get(SWING_STORE_META_KEY) || '[0, []]',
+    storage.get(SWING_STORE_META_KEY) || '[-1, []]',
```

Comment:
What's this for? Everything else makes sense to me, but this implies a fencepost-like change in behavior, and I didn't see an obvious place in the rest of the PR that would match.

Reply:
The fencepost was checking the DB state in the first block's commit (the first set of `BEGIN_BLOCK`...`END_BLOCK` transactions) vs. in the zeroth block's commit (upon init genesis, when the new `END_BLOCK` in `genesis.go` is dispatched).